app/register/page.tsx

```diff
@@ -1,228 +1,21 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-'use client';
-import React, { JSX, useEffect } from 'react';
-import { useRouter } from 'next/navigation';
-import { Input } from '@/components/Input/Input';
-import Logo from '@/components/Logo/Logo';
-import { Button } from '@/components/Button/Button';
-import { registerAccount } from '@/api/apiFunctions';
-import logo from '/public/assets/logo-colored-outline.svg';
-import { useAuthContext } from '@/context/AuthContextProvider';
-import LinkCustom from '@/components/LinkCustom/LinkCustom';
-import { z } from 'zod';
-import { Control, useForm, useWatch, SubmitHandler } from 'react-hook-form';
-import { zodResolver } from '@hookform/resolvers/zod';
-import {
-  Form,
-  FormControl,
-  FormField,
-  FormItem,
-  FormMessage,
-} from '../../components/Form/Form';
+import { JSX } from 'react';
+import { Metadata } from 'next';
+import Register from './Register';
```

Comment:
Could you please explain why the register page were separated out into a client and server component :)